src/main/java/uk/gov/hmcts/reform/professionalapi/controller/SuperController.java

```diff
@@ -505,4 +511,31 @@ public String getUserEmailFromHeader() {
 
         return userEmail;
     }
+
+    protected void deleteMultipleAddressOfGivenOrganisation(DeleteMultipleAddressRequest deleteRequest,
+                                                            String orgId) {
+        Set<String> addressIds = deleteRequest.getAddressId();
+
+        if (ObjectUtils.isEmpty(addressIds)) {
+            throw new InvalidRequest(ERROR_MSG_REQUEST_IS_EMPTY);
+        }
+        if (addressIds.contains(null) || addressIds.contains(EMPTY)
```

Comment:
does `addressIds.stream().anyMatch(StringUtils::isBlank)` not cover these conditions as well`addressIds.contains(null) || addressIds.contains(EMPTY)`?

Reply:
first one is check set is empty or second one is for set have null or empty as an element value.